samples/VanillaJSTestApp/server.js

```diff
@@ -19,7 +19,7 @@ app.use("/dist", express.static(path.join(__dirname, "../../lib/msal-core/dist")
 
 // Set up our one route to the index.html file.
 app.get('*', function (req, res) {
-    const reqPath = req.path === "/" ? "/index.html" : req.path;
+    const reqPath = req.path === "/" ? "/index_blankPageRedirectUri.html" : req.path;
```

Comment:
I don't agree with this change, the existing `index.html` should still be the default.

Reply:
Is it possible to set up different routes? That way we don't have to keep restarting the server to test different use cases.

Reply:
Well yeah we might as well do routes soon @pkanher617, I will take it up as a separate PR, not with this one.

@jasonnutter Yes, the default should remain `index.html`, changed now.